src/js/core.js

```diff
@@ -231,6 +231,21 @@
         }, this);
     }
 
+    function offDisabledElements(elements) {
+        // Should be checked that it exists outside the func.
+        elements.forEach(function (element) {
```

Comment:
Once you merge the latest master into your branch, you'll likely want to move this logic into `initElement`.  It's a function that will be called for every editor element during initialization and when elements are added dynamically.